solutions/silver/usaco-645.mdx

```diff
@@ -112,6 +113,100 @@ int main() {
 }
 ```
 </CPPSection>
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class SplittingTheField {
+	\```
+	    // returns updated min/maxs when considering a new coordinate
+	    public static Pair
+	    upd(Pair a, long b) {
+		return new Pair(Math.min(a.first, b), Math.max(a.second, b));
+	}
```

Comment:
wait is the [ \\``` ] a mistake or am i high

Reply:
it was there when i found it, i got rid of it -> build failed -> i put it back -> build failed again